ogc/common/core/templates/conformance.go.html

```diff
@@ -2,43 +2,43 @@
 {{define "content"}}
 <hgroup class="main">
     <h2 class="title">Conformance</h2>
```

Comment:
Zou dit ook wijzigen naar "Conformiteit voor {{ .Config.Title }}" zodat het gelijk is met de JSON representatie

Reply:
Goed idee!